stabilizer.py

```diff
@@ -27,6 +27,7 @@ async def set(self, channel, iir):
         assert "\n" not in s
         logger.debug("send %s", s)
         self.writer.write(s.encode("ascii") + b"\n")
+        self.writer.write_eof()
```

Comment:
Why write EOF here? We use the newline as "end-of-message" indications - this shouldn't be necessary.

Reply:
Stabilizer should not require the TX channel to be closed to be able to send a reply. That's the root of the problem here.